clar2wasm/src/words/consensus_buff.rs

```diff
@@ -1,13 +1,16 @@
-use clarity::vm::types::MAX_VALUE_SIZE;
-use walrus::ir::InstrSeqType;
+use clarity::vm::types::{TypeSignature, MAX_VALUE_SIZE};
+use walrus::ir::{BinaryOp, InstrSeqType};
 
 use super::Word;
-use crate::wasm_generator::ArgumentsExt;
+use crate::wasm_generator::{
+    add_placeholder_for_clarity_type, clar2wasm_ty, drop_value, ArgumentsExt, GeneratorError,
+    WasmGenerator,
+};
 
 #[derive(Debug)]
-pub struct ToConsensusBuf;
+pub struct To;
 
-impl Word for ToConsensusBuf {
+impl Word for To {
```

Comment:
I think using the full name makes more sense, especially in the `From` case, since it collides with a standard library trait.

Reply:
ok, that's fair. I will update this.